src/main/java/de/dhbw/karlsruhe/Main.java

```diff
@@ -1,16 +1,19 @@
 package de.dhbw.karlsruhe;
 
+import de.dhbw.karlsruhe.services.MenuDialogService;
 import de.dhbw.karlsruhe.services.StartUpDialogService;
 
 public class Main {
 
 	public static void main(String[] args) {
 		StartUpDialogService startUpDialogService = new StartUpDialogService();
+        MenuDialogService menuDialogService = new MenuDialogService();
```

Comment:
Formatting?

Reply:
In meiner IDE sieht es richtig aus, möglicherweise Anzeigefehler?